src/components/carousel/carousel.js

```diff
@@ -527,14 +529,14 @@ export default {
     }
     // Touch support event handlers for environment
     if (!this.noTouch && hasTouchSupport) /* istanbul ignore next: JSDOM doesn't support touch events */ {
-      // Attach appropriate listeners
+      // Attach appropriate listeners (passsive mode)
       if (hasPointerEvent) {
-        on.pointerdown = this.touchStart
-        on.pointerup = this.touchEnd
+        on['&pointerdown'] = this.touchStart
```

Comment:
Why is there a `&` prefixed to the event names?

Reply:
`&` is the render function version of the event modifier `.passive` (https://vuejs.org/v2/guide/render-function.html#Event-amp-Key-Modifiers)

Reply:
Thanks for pointing out!

Reply:
No prob!